samples/highcharts/demo/annotates-tsla-stock-price/demo.js

```diff
@@ -0,0 +1,232 @@
+const commonOptions = {
+    api: {
+        url: 'https://demo-live-data.highcharts.com',
+        access: {
+            url: 'https://demo-live-data.highcharts.com/token/oauth',
+            token: 'token'
+        }
+    }
+};
+
+const TeslaISIN = 'US88160R1014';
+
+// eslint-disable-next-line no-undef
+const connector = new HighchartsConnectors.Morningstar.TimeSeriesConnector({
+    ...commonOptions,
+    series: {
+        type: 'Price'
+    },
+    securities: [
+        {
+            id: TeslaISIN,
+            idType: 'ISIN'
+        }
+    ],
+    startDate: '2017-01-01',
+    endDate: '2024-01-01',
+    currencyId: 'EUR'
+});
+
+(async () => {
+    await connector.load();
+
+    const cols = connector.table.getColumns();
+
+    const name = Array.from(Object.keys(cols).filter(k => k !== 'Date'))[0];
+    const data = cols[name].map((value, i) => [cols.Date[i], value]);
+
+    Highcharts.chart('container', {
+        chart: {
+            type: 'area',
+            zooming: {
+                type: 'x'
+            },
+            panning: true,
+            panKey: 'shift',
+            scrollablePlotArea: {
+                minWidth: 600
```

Comment:
Recent data is more relevant for readers than the older data, so it's better to start the chart scrolled to the right.
```suggestion
                minWidth: 600,
                scrollablePositionX: 1
```